lib/main.dart

```diff
@@ -1,44 +1,28 @@
 import 'package:flutter/material.dart';
 
+import 'src/config/route/app_routes.dart';
+import 'src/core/util/route_constants.dart';
+
 void main() {
-  runApp(MyApp());
+  runApp(const MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
+  const MyApp({Key? key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
-      ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+      theme: ThemeData.dark(),
+      onGenerateRoute: AppRoutes.generateRoute,
+      initialRoute: RouteConstants.splashRoute,
     );
   }
 }
 
 class MyHomePage extends StatefulWidget {
```

Comment:
maybe changing the name to HomePage could be better, so it doesn't sounds like the default page